### ISSUE TITLE
Unable to setup OIDC Workload Identity Provider for Workload Pool

### ISSUE THREAD

**Issue by PranavNair0001** [google-github-actions/auth#201]
### TL;DR

As per the step 7 in README, a Workload Identity Provider isn't getting created, both from gcloud CLI as well as Pantheon UI.
In fact, the issuer-uri violates the organization policy, and thereby Precondition fails.

### Expected behavior

Successful creation of Workload Identity Provider

### Observed behavior

ERROR: (gcloud.iam.workload-identity-pools.providers.create-oidc) FAILED_PRECONDITION: Precondition check failed.
- '@type': type.googleapis.com/google.rpc.PreconditionFailure
  violations:
  - description: "Org Policy violated for value: 'https://token.actions.githubusercontent.com'."
    subject: orgpolicy:projects/project-step-pranav/locations/global/workloadIdentityPools/github-runner
    type: constraints/iam.workloadIdentityPoolProviders
- '@type': type.googleapis.com/google.rpc.DebugInfo
  detail: |-
    [ORIGINAL ERROR] generic::failed_precondition: The request has violated one or more Org Policies. Please refer to the respective violations for more information.
    com.google.apps.framework.request.StatusException: <eye3 title='FAILED_PRECONDITION'/> generic::FAILED_PRECONDITION: The request has violated one or more Org Policies. Please refer to the respective violations for more information. [google.rpc.error_details_ext] { details { [type.googleapis.com/google.rpc.PreconditionFailure] { violations { type: "constraints/iam.workloadIdentityPoolProviders" subject: "orgpolicy:projects/project-step-pranav/locations/global/workloadIdentityPools/github-runner" description: "Org Policy violated for value: \'https://token.actions.githubusercontent.com\'." } } } }

### Action YAML

```yaml
Irrelevant.

Command issued: 

$ gcloud iam workload-identity-pools providers create-oidc "github-runner-workload-provider" --project=project-step-pranav --location="global" --workload-identity-pool="github-runner" --display-name="GitHub Runner Pool Provider" --attribute-mapping="google.subject=assertion.sub,attribute.actor=assertion.actor,attribute.repository=assertion.repository" --issuer-uri="https://token.actions.githubusercontent.com"
```
```


### Log output

_No response_

### Additional information

A similar error was faced when trying to create from Pantheon UI as well.
Further, tried the command from the following Google Cloud blog too: https://cloud.google.com/blog/products/identity-security/enabling-keyless-authentication-from-github-actions
Didn't work.

**Comment by PranavNair0001** [comment]
My project is under 'google.com' organization. Could you please guide towards necessary links to allow the usage of the said issuer-uri

**Comment by PranavNair0001** [comment]
Resolved
